lib/mix/lib/mix/tasks/new.ex

```diff
@@ -445,8 +445,8 @@ defmodule Mix.Tasks.New do
     use ExUnit.Case
     doctest <%= @mod %>
 
-    test "the truth" do
-      assert 1 + 1 == 2
+    test "salutates the world" do
```

Comment:
I think the verb you are looking for might be `salutes`... or you could go with the more informal, `greets`

Thanks for this change, it's a great idea!